neuraxle/steps/loop.py

```diff
@@ -151,19 +151,20 @@ def __init__(self, wrapped: BaseStep, copy_op=copy.deepcopy, cache_folder_when_n
         BaseStep.__init__(self)
         MetaStepMixin.__init__(self, wrapped)
         ForceHandleOnlyMixin.__init__(self, cache_folder_when_no_handle)
+        self.savers.append(TruncableJoblibStepSaver())
```

Comment:
We don't have self.steps_as_tuple nor a refresh method. It seems like we're going to have problems at some points with those steps_as_tuple that may need a refresh or something at some point.